tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/Configs/ConfigRepository.cs

```diff
@@ -62,11 +63,10 @@ public static SecureString GetEncryptedPassword(string plainTextPassword)
         /// <param name="clientUrl"></param>
         public void UploadConfigFileToSPO(string filePath, string clientUrl)
         {
-            using (ClientContext clientContext = new ClientContext(generalSettings.CentralRepositoryUrl))
+            using (ClientContext clientContext = spoAuthorization.GetClientContext(clientUrl))
             {
-                //Connec to sharepoint using admin username and password
-                SecureString password = GetEncryptedPassword(generalSettings.AdminPassword);
-                clientContext.Credentials = new SharePointOnlineCredentials(generalSettings.AdminUserName, password);

```

Comment:
Removed the dependency for user name and password and used oAuth and bearer token